analysis_functions.py

```diff
@@ -0,0 +1,86 @@
+from typing import Union
```

Comment:
Plik zrobiony głównie do wypróbowania pisania funkcji w "dobrej konwencji"
Jak najbardziej do pozbycia się, ale zapomniałam

Reply:
Nawet możesz to zostawić, no i na podstawie tego można zacząć te funkcje pisać. Ale ja bym to wrzuciła do jakiegoś folderu innego.

Reply:
Na system wrzuciłam notebooki do osobnego folderu

Reply:
Wrzuciłam tam też te funkcje do analizy danych
Później można to przenieść do innego folderu

Reply:
Myślę, że dobrym pomysłem byłoby przemianowanie folderu "data analysis" na "data_analysis", prawdopodobnie nam by to nic nie psuło, ale mi na innych przedmiotach zdarzało się mieć problemy przez spacje w ścieżkach do plików